test/workers/suspend_inactive_accounts_worker_test.rb

```diff
@@ -53,8 +57,8 @@ class SuspendInactiveAccountsWorkerTest < ActiveSupport::TestCase
     assert already_deleted.scheduled_for_deletion?
   end
 
-  test 'it does not perform for on-prem' do
-    ThreeScale.config.stubs(onpremises: true)
+  test 'it does not perform for unless it has the valid configuration' do
```

Comment:
for what?

Reply:
### Right now in the master branch
SaaS has the feature to automatically suspend and delete accounts based in the values of some constants. On-prem does not have this feature at all.
### What we want to achieve with this PR
By default nobody has this feature unless they configure those values. So if they don't configure them, the feature is disabled, if they configure them correctly, it is enabled, and if they try to configure it but it is something wrong in the configuration, we decided to just show a warning message and have tis feature disabled, because if we raise and error at that time, it means it wouldn't boot at all.

Reply:
I was noting the wording, the correctness of the sentence...

Reply:
aaaah loooool right 👍 I hadn't noticed 😂 I will change it 😉